Gemfile

```diff
@@ -52,42 +35,26 @@ gem "fast_gettext",                   "~>1.2.0"
 gem "gettext_i18n_rails",             "~>1.7.2"
 gem "gettext_i18n_rails_js",          "~>1.1.0"
 gem "hamlit",                         "~>2.7.0"
-gem "htauth",                         "2.0.0",         :require => false
 gem "inifile",                        "~>3.0",         :require => false
-gem "jbuilder",                       "~>2.5.0" # For the REST API
 gem "manageiq-api-client",            "~>0.1.0",       :require => false
 gem "manageiq-network_discovery",     "~>0.1.1",       :require => false
-gem "manageiq-smartstate",            "~>0.1.1",       :require => false
 gem "mime-types",                     "~>2.6.1",       :path => "mime-types-redirector"
 gem "more_core_extensions",           "~>3.2"
 gem "nakayoshi_fork",                 "~>0.0.3"  # provides a more CoW friendly fork (GC a few times before fork)
 gem "net-ldap",                       "~>0.14.0",      :require => false
 gem "net-ping",                       "~>1.7.4",       :require => false
 gem "net-ssh",                        "=3.2.0",        :require => false
-gem "open4",                          "~>1.3.0",       :require => false
 gem "openscap",                       "~>0.4.3",       :require => false
-gem "ovirt-engine-sdk",               "~>4.1.4",       :require => false # Required by the oVirt provider
-gem "ovirt_metrics",                  "~>1.4.1",       :require => false
-gem "pg-pglogical",                   "~>1.1.0",       :require => false
-gem "puma",                           "~>3.3.0"
 gem "query_relation",                 "~>0.1.0",       :require => false
 gem "rails",                          "~>5.0.2"
 gem "rails-i18n",                     "~>5.x"
-gem "responders",                     "~>2.0"
 gem "rest-client",                    "~>2.0.0",       :require => false
 gem "ripper_ruby_parser",                              :require => false
-gem "ruby-dbus" # For external auth
 gem "ruby-progressbar",               "~>1.7.0",       :require => false
 gem "rubyzip",                        "~>1.2.1",       :require => false
-gem "rufus-scheduler",                "~>3.1.3",       :require => false
 gem "rugged",                         "~>0.25.0",      :require => false
-gem "secure_headers",                 "~>3.0.0"
 gem "simple-rss",                     "~>1.3.1",       :require => false
 gem "snmp",                           "~>1.2.0",       :require => false
-gem "sshkey",                         "~>1.8.0",       :require => false
-gem "thin",                           "~>1.7.0",       :require => false
```

Comment:
Very important: thin is completely gone since it doesn't support our async notifications using websockets.